onnxruntime/core/providers/cpu/tensor/concat.cc

```diff
@@ -36,19 +39,19 @@ Status ConcatBase::PrepareForCompute(OpKernelContext* ctx, int input_count, Prep
   for (int index = 0; index < input_count; index++) {
     tensor_pointer = ctx->Input<Tensor>(index);
     if (tensor_pointer == nullptr) return Status(common::ONNXRUNTIME, common::FAIL, "input count mismatch");
-    concat_axis_size += tensor_pointer->Shape()[int(axis_)];
+    concat_axis_size += tensor_pointer->Shape()[int(axis)];
```

Comment:
why cast axis to in?

Reply:
Because the operator[] for Shape() didn't take int64_t initially so it used to give errors. That was back when it was first written. I can change it and see if you want me to.